switch/enet/PKGBUILD

```diff
@@ -0,0 +1,39 @@
+pkgname=switch-enet
+pkgver=r252.e2ef839
+pkgrel=1
+pkgdesc='ENet reliable UDP networking library (adapted for Nintendo Switch homebrew development)'
+arch=('any')
+url='https://github.com/lsalzman/enet'
+license=('MIT')
+options=(!strip libtool staticlibs)
+makedepends=('git' 'switch-pkg-config' 'devkitpro-pkgbuild-helpers')
+source=(${pkgname}::"git+https://github.com/lsalzman/enet.git#commit=e2ef83927d1626d6ee479742343a3011333fdfdd")
```

Comment:
This can be done from a release tag now, just need to name the tarball we download, like this.

`${pkgname}-${pkgver}.tar.gz::${url}/archive/v${pkgver}.tar.gz`